hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/om-overview.html

```diff
@@ -49,6 +45,37 @@ <h2>Status</h2>
     </tbody>
 </table>
 
+<h4 ng-show="$ctrl.overview.jmx.RatisRoles[0] == 'Exception: Not a Leader' || $ctrl.overview.jmx.RatisRoles[0] == 'IO-Exception Occurred' || $ctrl.overview.jmx.RatisRoles[0] == 'Ratis Disabled'">Exception: {{$ctrl.overview.jmx.RatisRoles[0]}}</h4>
+<div ng-hide="$ctrl.overview.jmx.RatisRoles[0] == 'Exception: Not a Leader' || $ctrl.overview.jmx.RatisRoles[0] == 'IO-Exception Occurred' || $ctrl.overview.jmx.RatisRoles[0] == 'Ratis Disabled'">
```

Comment:
I think it would be better if we could check the size of `RatisRoles` and do the visualisation based on that:

- If we have one element in `RatisRoles` and inside that nested list only one more (we can check `$ctrl.overview.jmx.RatisRoles[0].length`), we can assume that we have an exception, that we can show as one line and no table.
- If we have one element in `RatisRoles`, but we have more inside that, we can say that we have only one OM. In that case we don't want to show anything (not even the title of the table), as all the information is already there in the _Status_ above this. We only want to show information if we have OM HA.
- If we have multiple elements in `RatisRoles`, then we can think the we have multiple OMs, so we can show that in the table as it is already happening.